packages/hadron-document/src/object-generator.ts

```diff
@@ -28,15 +28,25 @@ function maybeDecorateWithDecryptedKeys(
 }
 
 /** Used to represent missing values, i.e. non-existent fields. */
-const DoesNotExist = Symbol('DidNotExist');
+const DoesNotExist = Symbol('DoesNotExist');
```

Comment:
Purely cosmestic, I think I just forgot to change the name while working on #3239.